tests/golos.vesting_tests.cpp

```diff
@@ -145,15 +177,23 @@ BOOST_FIXTURE_TEST_CASE(convert, golos_vesting_tester) try {
     auto start_convert = [&](auto amount, auto precision, auto name) {
         return vest.convert_vesting(N(sania), N(sania), asset(amount, symbol(precision, name)));
     };
-    BOOST_CHECK_EQUAL(err.unknown_asset, start_convert(1e6, _vesting_precision, "GLS"));
+    BOOST_CHECK_EQUAL(err.vesting_params, start_convert(1e6, _vesting_precision, "GLS"));
     BOOST_CHECK_EQUAL(err.wrong_precision, start_convert(1e5, _vesting_precision-1, _token_name));
     BOOST_CHECK_EQUAL(err.wrong_precision, start_convert(1e7, _vesting_precision+1, _token_name));
 
     BOOST_TEST_MESSAGE("--- succeed if start converting with correct asset");
-    BOOST_CHECK_EQUAL(success(), vest.convert_vesting(N(sania), N(sania), _one_per_interval));
-    auto steps = cfg::vesting_withdraw.intervals;
+
+    BOOST_CHECK_EQUAL(true, t.is_object());
+
+    auto obj = mvo(t);
+    asset one_per_interval = vest.make_asset(obj["vesting_withdraw_params"]["intervals"].as<uint32_t>());
+    BOOST_CHECK_EQUAL(success(), vest.convert_vesting(N(sania), N(sania), one_per_interval));
+
+    auto steps = obj["vesting_withdraw_params"]["intervals"].as<uint32_t>();
     BOOST_CHECK_EQUAL(vest.get_convert_obj(N(sania))["number_of_payments"], steps);
 
+    auto blocks_in_interval = obj["vesting_withdraw_params"]["interval_seconds"].as<uint32_t>();
```

Comment:
It looks like these variables are needed in several tests. I think it is necessary to render the code for the formation of these separately.